### PR TITLE
Kerberos / GSSAPI docs initial update based on customer engagement

### PR DESCRIPTION
**Pull request by keith-mcclellan**
This format is all wrong but it has the additional info required to get started

### CODE REVIEW

File: v19.2/gssapi_authentication.md [hunk 1]
```diff
@@ -103,16 +132,43 @@ The `cockroach sql` shell does not yet support GSSAPI authentication. You need t
 {{site.data.alerts.end}}
 
 1. Install and configure your Kerberos client.
+
+In CentOS/RHEL - `yum install krb5-user`
```
Comment (Amruta-Ranade) [inline v19.2/gssapi_authentication.md]: Is this specific for CentOS/RHEL? Do we have instructions for other systems?

Comment (keith-mcclellan) [reply v19.2/gssapi_authentication.md]: should be `apt-get install krb5-user`

**Comment by jseldess** [comment]
@Amruta-Ranade, @keith-mcclellan, any updates on this PR?

File: v19.2/gssapi_authentication.md (hunk 1 above)
Comment (Amruta-Ranade) [reply v19.2/gssapi_authentication.md]: Done.

**Pull request merged**